extensions/wikia/WikiaMobile/WikiaMobileService.class.php

```diff
@@ -236,6 +240,12 @@ private function handleToc(){
 		$this->response->setVal( 'toc', $toc );
 	}
 
+	private function handleWikiaWidgets() {
+		if ( $this->isArticleView() ) {
+			$this->jsExtensionPackages[] = 'wikiamobile_widget_iframe_unwrapper';
```

Comment:
maybe you can use `$this->response->addAsset('wikiamobile_widget_iframe_unwrapper')` this way you don't have to create hook etc.